3.12.1 installation failure, patch not found

And it seems that the patch has really dissappeared. @hhugo  ?

````
=-=- Installing compiler 3.12.1 -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[default.comp] http://caml.inria.fr/pub/distrib/ocaml-3.12/ocaml-3.12.1.tar.gz downloaded
[ERROR] curl: code 404 while downloading https://github.com/hhugo/ocaml/compare/3.12.1...fix-3.12.1.diff
Fatal error:
# opam-version    1.2.2 (391297300524717e1475deaea37bca287c7f5e36)
# os              linux
curl: code 404 while downloading https://github.com/hhugo/ocaml/compare/3.12.1...fix-3.12.1.diff
````

--- a/compilers/3.12.1/3.12.1/3.12.1.comp
+++ b/compilers/3.12.1/3.12.1/3.12.1.comp
@@ -1,7 +1,6 @@
 opam-version: "1"
 version: "3.12.1"
 src: "http://caml.inria.fr/pub/distrib/ocaml-3.12/ocaml-3.12.1.tar.gz"
-patches: ["https://github.com/hhugo/ocaml/compare/3.12.1...fix-3.12.1.diff"]
 build: [
   ["./configure" "-prefix" prefix]
   [make "world"]

